modules/ocf/manifests/systemd/override.pp

```diff
@@ -7,21 +7,18 @@
 #     ExecStart=
 #     ExecStart=/path/to/my/thing
 define ocf::systemd::override(
-    $service,
+    $unit,
     $source = undef,
     $content = undef,
 ) {
-  ensure_resource('file', "/etc/systemd/system/${service}.service.d", {
+  ensure_resource('file', "/etc/systemd/system/${unit}.d", {
     ensure => directory,
   })
 
-  file { "/etc/systemd/system/${service}.service.d/${title}.conf":
-    source   => $source,
-    content  => $content,
-    notify   => [
-      Exec['systemd-reload'],
-      Service[$service],
-    ],
-    require  => Package['systemd-sysv'],
+  file { "/etc/systemd/system/${unit}.d/${title}.conf":
+    source  => $source,
+    content => $content,
+    notify  => Exec['systemd-reload'],
```

Comment:
It's up to the caller to notify any service that needs to be reloaded now. Is this too prone to programmer error? Are we worried about any potential race conditions, of `Service` being refreshed before `systemd-reload` happens?